nassl/ssl_client.py

```diff
@@ -95,6 +95,7 @@ def __init__(
         client_key_type: OpenSslFileTypeEnum = OpenSslFileTypeEnum.PEM,
         client_key_password: str = "",
         ignore_client_authentication_requests: bool = False,
+        name_indication: Optional[str] = None,
```

Comment:
Can you rename this to "server_name_indication"

Reply:
Done